invenio_app_rdm/__init__.py

```diff
@@ -17,6 +17,6 @@
 #
 # See PEP 0440 for details - https://www.python.org/dev/peps/pep-0440
 
-__version__ = "13.0.0b0.dev8"
+__version__ = "13.0.0b0.dev9"
```

Comment:
I am bit confused, wasn't dev9 already released? https://github.com/inveniosoftware/invenio-app-rdm/commit/d4088f6f551799ba328a47171d76aa349b189088

Reply:
Oh yes, I think the PR was older than that release, sorry. Updating.